### PR TITLE
Revival surgery no longer requires research

### PR DESCRIPTION
**Pull request by SomeguyManperson**
### Intent of your Pull Request

The revival surgery is stupid underused, it used to be important for fixing bodies past the defib timer but now that organs also decay it'd only be useful if you needed to revive someone in a scenario where you
A: have no cloning
B: have a mostly intact body, likely with decayed organs
C: have replacement organs
and D: have a full surgical setup and experimental research
by removing the technology requirement, it's now a ghetto revive usable if you can't get to a defib but have access to a stunbaton and some alcohol to even out lower surgery chances from using alternative tools.

#### Changelog

:cl:  
tweak: revival surgery is now a base surgery
/:cl:

### CODE REVIEW

File: code/modules/surgery/advanced/revival.dm [hunk 1]
```diff
@@ -14,7 +14,7 @@
 	requires_bodypart_type = 0
 
 /datum/surgery/advanced/revival/can_start(mob/user, mob/living/carbon/target)
```
Comment (monster860) [inline code/modules/surgery/advanced/revival.dm]: these type paths need to change to you fuck

**Review by monster860 — CHANGES_REQUESTED** [review]

**Comment by Hopekz** [comment]
I think if our rules changed for this to be considered a "new life" to make people "forget" this may work on our server.

**Comment by athena413** [comment]
sure

**Comment by alexkar598** [comment]
It fits the ability to be able to revive someone and get their memories of who killed them but at the cost of tech and resources, not a fan of this pr

**Pull request closed without merging**